src/utils/set.js

```diff
@@ -0,0 +1,4 @@
+const set = (updater, arg) =>
+  typeof updater === 'function' ? updater(arg) : updater
+
+export default set
```

Comment:
Are you gonna reuse it? Where?

Reply:
everywhere xD

all components will have set function that will work as this

see https://github.com/renatorib/react-powerplug/issues/6